### DISCUSSION TITLE
Pure CSS "go back to top" button?

### DISCUSSION THREAD

**Discussion by Meithal** [picocss/pico#392] [Ideas]
Is it possible to have a pure CSS "go back to top" button?

This article shows a solution for that https://moderncss.dev/pure-css-smooth-scroll-back-to-top/

What do you think about this feature, is it doable?

**Comment by utkarsh-glitch** [comment]
with only css, it would be hard to do it. that's why we use javascript. but there are some CSS geniuses who can pull this really well.

**Comment by Sathyaraj-dev** [comment]
Hi,

Yes, it's possible to do only with HTML & CSS. Please check this link for your reference. https://dev.to/atulcodex/scroll-to-top-button-with-just-html-css-1c2

**Comment by ricpastori** [comment]
The essential “back to top” behavior is already provided by HTML:

```html
<a href="#top">Back to top</a>
```

Whether that link should be fixed, sticky, always visible or revealed after scrolling is specific to the page layout. CSS-only techniques that conditionally reveal it tend to rely on layout tricks, while smooth scrolling also needs to respect reduced-motion preferences.

This could be a useful documentation recipe, but I don’t think it requires a component in Pico itself.
